apps/web/src/utils/clsxm.ts

```diff
@@ -9,4 +9,6 @@ import { twMerge } from 'tailwind-merge'
  */
 const clsxm = (...classes: ClassValue[]) => twMerge(clsx(...classes))
 
+export const cn = clsxm
```

Comment:
Add a comment as to why there is another export, here is my suggestion,
```suggestion
// An alias export to make the integration with shadnc/ui smoother
export const cn = clsxm
```